Index: package-lock.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- package-lock.json	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ package-lock.json	(revision 1fc2102ac3b22156454f45bbdaecd303fd86d259)
@@ -166,6 +166,23 @@
         "tslib": "^1.9.0"
       }
     },
+    "@angular/cdk": {
+      "version": "8.2.3",
+      "resolved": "https://registry.npmjs.org/@angular/cdk/-/cdk-8.2.3.tgz",
+      "integrity": "sha512-ZwO5Sn720RA2YvBqud0JAHkZXjmjxM0yNzCO8RVtRE9i8Gl26Wk0j0nQeJkVm4zwv2QO8MwbKUKGTMt8evsokA==",
+      "requires": {
+        "parse5": "^5.0.0",
+        "tslib": "^1.7.1"
+      },
+      "dependencies": {
+        "parse5": {
+          "version": "5.1.0",
+          "resolved": "https://registry.npmjs.org/parse5/-/parse5-5.1.0.tgz",
+          "integrity": "sha512-fxNG2sQjHvlVAYmzBZS9YlDp6PTSSDwa98vkD4QgVDDCAo84z5X1t5XyJQ62ImdLXx5NdIIfihey6xpum9/gRQ==",
+          "optional": true
+        }
+      }
+    },
     "@angular/cli": {
       "version": "8.3.12",
       "resolved": "https://registry.npmjs.org/@angular/cli/-/cli-8.3.12.tgz",
Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- package.json	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ package.json	(revision 1fc2102ac3b22156454f45bbdaecd303fd86d259)
@@ -19,6 +19,7 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~8.1.1",
+    "@angular/cdk": "^8.2.3",
     "@angular/common": "~8.1.1",
     "@angular/compiler": "~8.1.1",
     "@angular/core": "~8.1.1",
Index: src/app/app.component.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/app.component.html	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ src/app/app.component.html	(revision 975aff779c344b3a27b53f1d77ac5b72cdc7ba83)
@@ -1,5 +1,9 @@
 <!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
+  <div class="rally-card-container">
+    <rally-finance-card title="Accounts" value="$12,000" visibleItemsSize="3" [items]="accountItems"></rally-finance-card>
+    <rally-finance-card title="Bills" value="$12,000" visibleItemsSize="3" [items]="accountItems"></rally-finance-card>
+  </div>
   <h1>
     Welcome to {{ title }}!!!111!!
   </h1>
Index: src/app/app.module.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/app.module.ts	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ src/app/app.module.ts	(revision 975aff779c344b3a27b53f1d77ac5b72cdc7ba83)
@@ -3,10 +3,11 @@
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { RallyFinanceCardModule } from './rally-finance-card/rally-finance-card.module';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule],
+  imports: [BrowserModule, AppRoutingModule, RallyFinanceCardModule],
   providers: [],
   bootstrap: [AppComponent],
 })
Index: src/app/rally-finance-card/rally-finance-card.component.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/rally-finance-card/rally-finance-card.component.html	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ src/app/rally-finance-card/rally-finance-card.component.html	(revision 975aff779c344b3a27b53f1d77ac5b72cdc7ba83)
@@ -1,1 +1,15 @@
-<p>rally-finance-card-component works!</p>
+<div class="rally-finance-card-container">
+  <mat-card class="rally-finance-card">
+    <div class="category-container">
+      <span>{{title}}</span>
+      <span>{{value}}</span>
+    </div>
+    <div *ngFor="let item of items; let i = index;" class="subcategory-container">
+      <ng-container *ngIf="i < visibleItemsSize">
+        <span >{{item.title}}</span>
+        <span>{{item.subtitle}}</span>
+        <span>{{item.value}}</span>
+      </ng-container>
+    </div>
+  </mat-card>
+</div>
Index: src/app/rally-finance-card/rally-finance-card.component.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/rally-finance-card/rally-finance-card.component.ts	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ src/app/rally-finance-card/rally-finance-card.component.ts	(revision 975aff779c344b3a27b53f1d77ac5b72cdc7ba83)
@@ -24,6 +24,10 @@
 
   /** Items belonging to this category. */
   @Input() items: FinanceSubcategory[];
+
+  /** Number of items that are visible in the collapsed view of this component. */
+  @Input() visibleItemsSize: number;
+
   constructor() {}
 
   ngOnInit() {}
Index: src/app/rally-finance-card/rally-finance-card.module.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/rally-finance-card/rally-finance-card.module.ts	(revision 919dfcb793339a379a850da1765743e2678e1b5a)
+++ src/app/rally-finance-card/rally-finance-card.module.ts	(revision 1fc2102ac3b22156454f45bbdaecd303fd86d259)
@@ -6,5 +6,6 @@
 @NgModule({
   declarations: [RallyFinanceCardComponent],
   imports: [CommonModule, MatCardModule],
+  exports: [RallyFinanceCardComponent],
 })
 export class RallyFinanceCardModule {}
Index: src/app/app.component.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/app.component.scss	(date 1572025809000)
+++ src/app/app.component.scss	(date 1572029431000)
@@ -1,4 +1,13 @@
 h1 {
   background-color: blue;
-  font-size: 50px;   
-}
\ No newline at end of file
+  font-size: 50px;
+}
+
+.rally-card-container {
+  display: flex;
+  flex-direction: row;
+}
+
+rally-finance-card {
+  flex-grow: 1;
+}
Index: src/app/app.component.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/app.component.ts	(date 1572025809000)
+++ src/app/app.component.ts	(date 1572029431000)
@@ -7,4 +7,9 @@
 })
 export class AppComponent {
   title = 'rally';
+  accountItems = [
+    { title: 'Checking', subtitle: '********1234', value: 2215.13 },
+    { title: 'Home Savings', subtitle: '********4234', value: 8676.88 },
+    { title: 'Car savings', subtitle: '********9012', value: 900.13 },
+  ];
 }
Index: src/app/rally-finance-card/rally-finance-card.component.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/rally-finance-card/rally-finance-card.component.scss	(date 1572025809000)
+++ src/app/rally-finance-card/rally-finance-card.component.scss	(date 1572029431000)
@@ -1,0 +1,21 @@
+.rally-finance-card-container {
+  background-color: black;
+  color: white;
+  margin: 5px;
+}
+
+.rally-finance-card {
+  display: flex;
+  flex-direction: column;
+
+  .category-container {
+    display: flex;
+    flex-direction: column;
+    align-self: flex-start;
+  }
+
+  .subcategory-container {
+    align-self: flex-start;
+  }
+}
+
